docs/openshift.md

```diff
@@ -7,3 +7,20 @@ DirectPV runs under project `directpv` in Red Hat OpenShift. Project `directpv`
 ## Limitations
 * DirectPV does not support volume snapshot feature as per CSI specification. DirectPV is specifically meant for use cases like MinIO where the data availability and resiliency is taken care by the application itself. Additionally, with the AWS S3 versioning APIs and internal healing, snapshots is not a requirement.
 * DirectPV does not support `ReadWriteMany` volume access mode. The workloads using DirectPV run local to the node and are provisioned from local storage drives in the node. This allows the workloads to directly access data without any additional network hops, unlike remote volumes, network PVs, etc. The additional network hops may lead to poor performance and increases the complexity. With `ReadWriteOnce` access mode, DirectPV provides high performance storage for Pods.
+
+## SELinux in OpenShift:
+
+If you encounter the `relabel failed` error after executing the `suspend` or `resume` commands, you should set `spc_t` at the Tenant level Specification, as demonstrated below:
+
+```yaml
+kind: Tenant
```

Comment:
1. This is more specific to MinIO Operator, but DirectPV is meant for any container require local storage access. We would need to add documentation to generic use case as primary and extend it to MinIO server and MinIO Operator.
2. Does pod level setting `Pod.Spec.SecurityContext.SELinuxOptions.Type = spc_t`  work?

Reply:
^^ @cniackz

Reply:
@cniackz ^^ PTAL

Reply:
It make sense, let's make it generic